aspnetcore/blazor/debug.md

```diff
@@ -45,54 +44,117 @@ Debugging requires either of the following browsers:
 * Google Chrome (version 70 or later)
 * Microsoft Edge preview ([Edge Dev Channel](https://www.microsoftedgeinsider.com))
 
+## Enable debugging
```

Comment:
Adding `inspectUri` is only needed for debugging in our IDEs. If you just want to debug in the browser, then this isn't really required.

In addition to debugging in our IDEs, we still need cover how to debug in a browser using the browser dev tools.
1. Run a Debug build of the app in the Development environment
1. Hit <kbd>Shift-Alt-D</kbd>
1. Follow the instructions to restart the browser with remote debugging enabled if needed.
1. Use the browser dev tools to debug your app

Then we can cover debugging in an IDE